index.js

```diff
@@ -1,33 +1,44 @@
-var http, director, cool, bot, router, server, port;
+// Set the bot token from MeBots in .env under BOT_TOKEN
+let mebots = require('mebots');
+let bot = new mebots.Bot('your_bot_shortname_here', process.env.BOT_TOKEN);
 
-http        = require('http');
-director    = require('director');
-cool        = require('cool-ascii-faces');
-bot         = require('./bot.js');
+// Setup our web routing
+const express = require("express");
+const app = express();
 
-router = new director.http.Router({
-    '/' : {
-        post: bot.respond,
-        get: ping
-    }
+// Library to send POST requests simply
+const axios = require('axios')
+
+// Simple GET route to ping the server
+app.use(express.json())
+app.get("/", function(req, res) {
+  res.sendStatus(200)
 });
 
-server = http.createServer(function (req, res) {
-    req.chunks = [];
-    req.on('data', function (chunk) {
-        req.chunks.push(chunk.toString());
-    });
+// Use this route as the callback URL for the bot
+// Example: https://shadow.herokuapp.com/gm
+app.post('/gm', async (req, res)  => {
```

Comment:
Hmm, I don't really like this route naming. I usually use the `/` route, which is not ideal, but perhaps we should choose something more clear, for example `/receive`? `gm` is kind of ambiguous.